### DISCUSSION TITLE
Valet needs to be restarted each time i start my day

### DISCUSSION THREAD

**Discussion by whitespacecode** [laravel/valet#1251] [Q&A]
- Valet Version: 3.1.6
- PHP Version: 8.1.5

### Description:
Coming from macbook pro to apple m1 i noticed each time i start my day i need to enter the command to restart valet
`valet restart`

### Diagnosis
I don't know if it's because i'm on a m1 now or a setting i need to switch but i was used to just open the lid and start whatever.
Anything i can do to help diagnose this problem further ?

- Reshing of my development page i get 
```
DNS address could not be found. Diagnosing the problem.
DNS_PROBE_POSSIBLE
```
- restarting only nginx doesn't help 

- Restarting only dnsmasq does seem to fix my issue

**Comment by pintend** [comment]
i have the same thing, i think its related to network changes

**Reply by pryley** [reply]
Me too, on M1. This happens whenever the network or wifi connection loses internet access. I have tried manually restarting services and running `valet restart`, but it seems that the only _reliable_ way to fix it is to run `valet install` each time.

**Reply by pryley** [reply]
`valet restart` does not always fix this, but @JoeriTheGreat is correct in that only restarting dnsmasq does.

`sudo brew services restart dnsmasq`

This is happening almost daily for me, so I made an Alfred workflow to automate it. 

<img width="556" alt="image" src="https://user-images.githubusercontent.com/134939/181197049-b588a618-b303-4ca4-a7b5-155d349539b9.png">

**Comment by rtconner** [comment]
I have this same issue. And yeah I'm on Macbook M1 also.
